### ISSUE TITLE
Make the 404 / 403 page setting "translatable"

### ISSUE THREAD

**Issue by indigoxela** [backdrop/backdrop-issues#4881]
**Description of the need**

Currently the 404 and 403 page setting isn't translatable, so all languages on a multilingual site get the same path. Even it that page is a translated node.

**Proposed solution**

We could either add per-language settings if the language module is installed, or we could follow the node translation if the page is a node and translations exist.

**Alternatives that have been considered**

- A layout (single path) that uses the translated node as "existing content"
- Contrib module

Both don't seem ideal, as the former requires some knowledge of the layout system, the latter requires people to know about the module.

**Is there a Drupal or Backdrop contributed module that accomplishes this?**

i18n does that for D7 (with bad UX), but that feature requires the (unported / unportable) Variables module for i18n_variables.

**Additional information**

This popped up in the i18n (contrib) issue queue.

See [this comment](https://github.com/backdrop-contrib/i18n/issues/39#issuecomment-761548065) for the steps to do it with a layout. We can't assume, a Backdrop newbie is able to do this.

**Comment by klonos** [comment]
I agree that this should be handled properly in core. Not sure re implementation (so that we don't have code bloat etc.), but the expected behavior is that if the site has been configured to be multilingual, then the translated versions of the default English text shown for 403/404 should be shown. If the default 403/404 pages have been replaced by a node with translations, then the translated nodes should be shown (with a fall-back to the default language or source node).

**Comment by indigoxela** [comment]
For completeness: the original strings "The requested page "@path" could not be found." and "You are not allowed to access this page." *are* translatable via `t()`, this issue is about when people set a different path on admin/config/system/site-information. Things get difficult then.

**Comment by olafgrabienski** [comment]
> We could either add per-language settings if the language module is installed, or we could follow the node translation if the page is a node and translations exist.

The latter sounds good, it's more straightforward, in my opinion.
